elastic/beats
# Add ELU fields to metricbeat Kibana module.

This PR updates the Metricbeat `kibana` module with support for the new `event_loop_utilization` field added to Kibana in https://github.com/elastic/kibana/pull/153717. We plan to use this metric for autoscaling in serverless, so metricbeat needs to know about it.

## Checklist


## How to test this PR locally

- run kibana from source & ES from snapshot as explained in https://www.elastic.co/guide/en/kibana/master/development-getting-started.html
- pull down this PR, `cd metricbeat`
- `mage build`
- `./metricbeat modules enable kibana`
- uncomment the `elasticsearch.output` username/password lines in `metricbeat.yml` (`elastic:changeme`)
- update `./modules.d/kibana.yml`:
```yml
- module: kibana
  metricsets:
    - stats # be sure to enable stats metricset
  period: 10s
  hosts: ["localhost:5601"]
  basepath: "abc" # whatever your base path is
  username: "elastic"
  password: "changeme"
```
- `./metricbeat -e`
- create `metricbeat-*` data view in kibana
- you should now be seeing some log entries which contain `event_loop_utilization`
